src/main/xerus/monstercat/tabs/TabSettings.kt

```diff
@@ -83,6 +85,13 @@ class TabSettings: VTab() {
 			Settings.PLAYERSEEKBARHEIGHT.bind(valueProperty() as ObservableValue<out Double>)
 		})
 		
+		add(CheckBox("Enable Streamer Mode (skips copyright-unsafe tracks in the player)").bind(Settings.SKIPUNLICENSABLE)).apply {
+			APIConnection.connectValidity.listen { validity ->
+				isDisable = validity != ConnectValidity.GOLD
```

Comment:
why? is everything unlicensable without Gold?

Reply:
Yes :( it's an API limitation apparently. I'll try to ask Cody

Reply:
licensable = false if you're not logged in as gold.

Reply:
nevermind, just checked. my bad, reverted in 64a621fc0f287bcdc739580d3aa3353f5c1ed33c

Reply:
You can resolve